data/mods/gen1/conditions.ts

```diff
@@ -87,8 +87,9 @@ export const Conditions: {[id: string]: ModdedConditionData} = {
 		effectType: 'Status',
 		onStart(target) {
 			this.add('-status', target, 'frz');
+			delete target.volatiles['mustrecharge']?.duration;
```

Comment:
You should remove `duration` from `mustrecharge` directly, unless there exists another effect that suppresses "Must Recharge" but still makes it end early?